packages/qdrant-loader-mcp-server/src/qdrant_loader_mcp_server/search/enhanced/cross_document_intelligence.py

```
@@ -1366,43 +1773,421 @@ def _get_complementary_content_type_score(self, target_doc: SearchResult, candid
+            return 0.0
+            
+        # Calculate Jaccard similarity
+        intersection = len(words1 & words2)
+        union = len(words1 | words2)
+        
+        if union == 0:
+            return 0.0
+            
+        return intersection / union
+
+    async def detect_conflicts(self, documents: List[SearchResult]) -> ConflictAnalysis:
```

Ruff (0.12.2)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)